resources/measures/HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -652,17 +651,15 @@ def self.process_infiltration_for_unit(model, runner, obj_name_infil, infil, win
             ext_wall_area_building = (n_end_units * ext_wall_area_end) + (n_mid_units * ext_wall_area_mid)
             ext_wall_area_building = UnitConversions.convert(ext_wall_area_building, 'm^2', 'ft^2')
           end
-          a_o = building.SLA * building_ag_ffa * (unit_ag_ext_wall_area / ext_wall_area_building) # Effective Leakage Area (ft^2) - Unit
+          a_o = building.SLA * building_ffa * (unit_ag_ext_wall_area / ext_wall_area_building) # Effective Leakage Area (ft^2) - Unit
         end
       else # SFD
         num_floors = building.stories
-        a_o = building.SLA * building.ag_ffa * (unit_ag_ext_wall_area / building.ag_ext_wall_area) # Effective Leakage Area (ft^2) - Unit
-        building_ag_ffa = building.ag_ffa
-        ext_wall_area_building = building.ag_ext_wall_area
+        a_o = building.SLA * unit_ffa * (unit_ag_ext_wall_area / building.ag_ext_wall_area) # Effective Leakage Area (ft^2) - Unit
```

Comment:
Should the wall area terms not be limited to "above grade" as well?

Reply:
Maybe that would be less confusing, but they give the same answer. Since this code will be getting replaced, it didn't seem worth the effort.